includes/class-wc-payments-onboarding-service.php

```diff
@@ -1024,4 +1031,66 @@ public function maybe_add_test_drive_settings_to_new_account_request( array $arg
 
 		return $args;
 	}
+
+	/**
+	 * Updates the enabled payment methods for a gateway.
```

Comment:
Please add more details to the docs about where these capabilities come from, what their structure is, etc.

Reply:
Included an explanation with inline documentation.